Add some tools and client libraries (#6)
appuio/appcat-docs: docs/modules/ROOT/pages/object-storage/usage.adoc

= How to Use an AppCat Object Bucket

> == Usage Example within APPUiO

Reviewer: ```suggestion
== Usage Example within an APPUiO Cloud Zone
```

----
<1> Values copied from the secret's `ENDPOINT` and `BUCKET` fields.
<2> Directly referencing the fields in the generated secret `objectbucket-creds`.

== List of Clients and Libraries

> If you want to inspect the object buckets or access it outside of APPUiO you can use one of these client tools:

Reviewer: ```suggestion
If you want to inspect the object buckets or access it outside of APPUiO you can use one of these client tools:

```

* https://cyberduck.io/
* https://rclone.org/
* https://s3tools.org/s3cmd
* https://filezillapro.com/docs/v3/cloud/configure-filezilla-pro-to-connect-to-s3/

> For client libraries to be used in your project:
> go: https://minio-go.min.io/
> javascript: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/index.html
> python: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html

Reviewer: ```suggestion
The following client libraries are a suggestion:

* Go: https://minio-go.min.io/[MinIO Go Client SDK]
* JavaScript: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/index.html[AWS SDK for JavaScript v3]
* Python: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html[Boto3]
```
